bin/zkServer.sh

```diff
@@ -229,9 +229,9 @@ version)
     ;;
 restart)
     shift
-    "$0" stop ${@}
+    "$0" stop "${@}"
     sleep 3
-    "$0" start ${@}
+    "$0" start "${@}"
```

Comment:
FWIW, the braces aren't doing anything here, and `"${@}"` could be replaced by `"$@"`